### PR TITLE
Discovery/multiple routes

### PR DESCRIPTION
**Pull request by ryma2fhir**
First example to show how to order Paracetamol either orally or via IV, but not both. 

Uses MedicationRequest x2, RequestGroup & MedicationDispense

### CODE REVIEW

**Comment by ryma2fhir** [comment]
All examples are now valid.

File: examples/UKCore-MedicationRequest-ParacetamolIV-Example.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+			<code value="acute"/>
+			<display value="Short course (acute) therapy"/>
+		</coding>
+	</courseOfTherapyType>
+	<dosageInstruction>
+		<text value="Paracetamol 500mg, four times a day."/>
+		<timing>
+			<repeat>
+				<frequency value="4"/>
+				<period value="24"/>
```
Comment (APageNHS) [inline examples/UKCore-MedicationRequest-ParacetamolIV-Example.xml:38]: Does this period also need to change

**Review by APageNHS — APPROVED** [review]

File: examples/UKCore-RequestGroup-Paracetamol-Example.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<RequestGroup xmlns="http://hl7.org/fhir">
+	<id value="UKCore-RequestGroup-Paracetamol-Example"/>
+	<status value="completed"/>
+	<intent value="order"/>
+	<priority value="routine"/>
+	<code>
+		<coding>
+			<system value="https://dmd.nhs.uk"/>
```
Comment (APageNHS) [inline examples/UKCore-RequestGroup-Paracetamol-Example.xml:8]: This needs to be the snomed ct namingsystem

Comment (ryma2fhir) [reply examples/UKCore-RequestGroup-Paracetamol-Example.xml]: done

**Review by APageNHS — APPROVED** [review]

**Pull request merged**